pages/common/pcapfix.md

```diff
@@ -0,0 +1,24 @@
+# pcapfix
+
+> Prepair pcap and pcapng files.
+> More information: <https://f00l.de/pcapfix/>.
+
+- Repair a pcap/pcapng file::
```

Comment:
Thanks for the hint! However, the `--deep-scan` option says 262144 bytes. Do you know what this refers to?

Reply:
The README says 65536, did I get it wrong, is it outdated or something?
> Deep Scan Option
> ................
> 
> In classic pcap files, pcapfix will only scan the first 65536 bytes (maximum
> packet length) for a proper first packet. If you want to force packet detection
> even above this limit (e.g. because your file has been heavily destroyed) you
> can use the deep scan option (-d).
> 
> This option is not necessary for pcapng files because the whole file is arranged
> in blocks that are 'unlimited' by default. In result pcapfix will always scan
> the whole pcapng file for further blocks.

Reply:
The way I see it, the information on the man page is contradictory. I have created an issue about this: https://github.com/Rup0rt/pcapfix/issues/35

Reply:
https://github.com/Rup0rt/pcapfix/blob/devel/pcapfix.1#L24

> This option will force pcapfix to scan for any packet inside the whole file (instead of the first 262144 (max snaplen) bytes only).

Reply:
It is 262144, see https://github.com/Rup0rt/pcapfix/issues/35#issuecomment-19425590847 for more details

Reply:
I have added the note. Thanks for the research!